apps/nextjs-website/src/app/auth/sign-up/page.tsx

```diff
@@ -133,14 +119,17 @@ const SignUp = () => {
         >
           <Grid item xs={isSmallScreen ? 1 : 5}>
             <Typography variant='h6' mb={4} mt={isSmallScreen ? 10 : 0}>
-              {signUp.whyCreateAccount}
+              {signUp('whyCreateAccount')}
             </Typography>
-            {signUp.advantages.map((advantage, index) => {
+            {advantages.map((advantage, index) => {
```

Comment:
Can be simplified as follows?

```suggestion
            {singUpAdvantages.map((advantage, index) => {
```